TimedArithmeticQuiz/src/App.jsx

```diff
@@ -0,0 +1,66 @@
+import { useCallback, useEffect, useRef, useState } from "react"
+import { QuizHeader } from "./components/QuizHeader"
+import { quizConfig } from "./data/quizConfig"
+import { Question } from "./components/Question";
+import { Quiz } from "./models/Quiz";
+import { QuestionHeader } from "./components/QuestionHeader";
+import { QuizComplete } from "./components/QuizComplete";
+
+
+function App() {
+  const [score, setScore] = useState(0);
+  const [quiz,] = useState(new Quiz(quizConfig.numberRange, quizConfig.operatrs ,quizConfig.totalQuestionCount));
+  const [currentQuestionIndex, setCurrrentQuestionIndex] = useState(0);
+  const [enteredAnswer, setEnteredAnswer] = useState("");
+
+  const initialized = useRef(false);
```

Comment:
Why are we using a ref for this?

Reply:
In strict mode useEffect is called twice. And as I am starting the quiz in useEffect it would be started twice. So, to prevent that I needed a bit to prevent the quiz from being started again.

So, here I am using useRef to presist this bit across renders.

Reply:
let's have a call about this. The explanation does not seem right.